README.md

```diff
@@ -1,21 +1,19 @@
-trimLeft and trimRight for ECMAScript
--------------------------------------
+# String.prototype.trimStart / String.prototype.trimEnd
 
-ES5 standardized `String.prototype.trim`. All major engines have also
-implemented corresponding `String.prototype.trimLeft` and `String.prototype.trimRight` functions - without any standard specification.
+ECMAScript proposal, specs, tests, and reference implementation for String.prototype.trimStart/trimEnd (plus **trimLeft/trimRight**).
 
-Since these are defacto standard across engines, I propose that these are standardized as is.
+This initial proposal was drafted by [@sebmarkbage](https://github.com/sebmarkbage) and the updated spec was drafted by [@evilpie](https://github.com/evilpie/) with input from [@ljharb](https://github.com/ljharb).
 
-## Bidi and Right-to-Left Strings
+This proposal is currently at [stage 1](https://github.com/tc39/ecma262) of the [process](https://tc39.github.io/process-document/).
 
-In [this discussion](https://esdiscuss.org/topic/string-prototype-trimright-trimleft) it was noted that the words `left` and `right` doesn't make sense for bidirectional and right-to-left languages. A more appropriate name for these functions would be `trimStart` and `trimEnd` which corresponds to the existing `startsWith` and `endsWith`.
+Designated TC39 reviewers: ?
```

Comment:
(the committee selects these once it reaches stage 2, which hopefully @sebmarkbage will propose in May)